src/app/ReadHandler.cpp

```diff
@@ -51,8 +51,17 @@ CHIP_ERROR ReadHandler::Init(InteractionModelDelegate * apDelegate, Messaging::E
     return err;
 }
 
-void ReadHandler::Shutdown()
+void ReadHandler::Shutdown(bool aAbort)
```

Comment:
any chance to have a signature using some 'flags enum' or similar instead of a bool. Without reading the signature, a code that reads:

```
foo->Shutdown(true);
bar->Shutdown(false);
```

is soewhat harder to read than something like:

```
foo->Shutdown(ReadHandler::kAbortCurrentExchange);
bar->Shutdown(ReadHandler::kKeepCurrentExchange);
```

Reply:
Done, thanks